.gitignore

```diff
@@ -1,6 +1,7 @@
 target/
 dist/
 **/*.rs.bk
+Cargo.lock
```

Comment:
Was this intentional? I think I prefer not to ignore it.

Reply:
Probably accidental, removed now.